src/apps/code-editor/src/app/components/FileActions/components/EditorActions/Save/Save.js

```diff
@@ -29,12 +28,14 @@ export function Save(props) {
   const metaShortcut = useMetaKey("s", onSave);
 
   return (
-    <Button type="save" onClick={onSave} disabled={saving}>
-      {saving ? (
-        <FontAwesomeIcon spin icon={faSpinner} />
-      ) : (
-        <FontAwesomeIcon icon={faSave} />
-      )}
+    <Button
+      variant="contained"
+      color="success"
+      onClick={onSave}
+      disabled={saving}
+      startIcon={saving ? <CircularProgress size="1rem" /> : <SaveIcon />}
+      sx={{ my: 0, mx: 0.5 }}
```

Comment:
Is there a reason you need to explicitly set my to 0? is it not already 0 by default?